.bazelrc

```diff
@@ -1,5 +1,6 @@
 common --experimental_enable_bzlmod
 
+#startup --output_user_root=C:/tmp
```

Comment:
You can put this in a `.bazelrc.local` file and since it's in `.gitignore` it won't get checked in.